contracts/misc/UiPoolDataProviderV2V3.sol

```diff
@@ -72,7 +72,7 @@ contract UiPoolDataProviderV2V3 is IUiPoolDataProviderV3 {
   {
     IAaveOracle oracle = IAaveOracle(provider.getPriceOracle());
     ILendingPool lendingPool = ILendingPool(provider.getLendingPool());
-    address[] memory reserves = lendingPool.getReservesList();
+    address[] memory reserves = lendingPool.getReservesList(address asset);
     AggregatedReserveData[] memory reservesData = new AggregatedReserveData[](reserves.length);
 
     for (uint256 i = 0; i < reserves.length; i++) {
```

Comment:
check where `getReservesList` is being called in this file and modify

Reply:
i modified this function to accept the pool address as this function is meant to return an address of active reserves(that is assets that are active in that pool address). this solution isn't the most ideal as i think it can be better especially in the LendingPool.sol implementation